File: 2024_12/leetcode2024-12-19.py
from typing import List

class Solution:
    def stableMountains(self, height: List[int], threshold: int) -> List[int]:
        n = len(height)
        res = []
        for i in range(1, n):
            if height[i-1] > threshold:
                res.append(i)
        return res

if __name__ == "__main__":
    s = Solution()
    height = [1,2,3,4,5]
    threshold = 2
    print(s.stableMountains(height, threshold))

File: leetcode2024/2024_12/leetcode2024-12-19.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    vector<int> stableMountains(vector<int>& height, int threshold) {
        vector<int> res;
        int n = height.size();
        for (int i=1; i<n; i++){
            if (height[i-1] > threshold){
                res.push_back(i);
            }
        }
        return res;
    }
};

int main(){
    Solution s;
    vector<int> height = {1,2,3,4,5};
    int threshold = 2;
    vector<int> res = s.stableMountains(height, threshold);
    for (int i : res){
        cout << i << " ";
    }
    cout << endl;
}
